spec/javascripts/health-care/components/form-elements/ErrorableInput.spec.jsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import ReactTestUtils from 'react-addons-test-utils';
+import SkinDeep from 'skin-deep';
+
+import ErrorableInput from '../../../../../_health-care/_js/_components/_form-elements/ErrorableInput';
+
+describe('<ErrorableInput>', () => {
+  describe('propTypes', () => {
+    let consoleStub;
+    beforeEach(() => {
+      consoleStub = sinon.stub(console, 'error');
+    });
+
+    afterEach(() => {
+      consoleStub.restore();
+    });
+
+    it('label is required', () => {
```

Comment:
I'm a little confused what this is testing exactly. Is this testing that if the label is not specified, that an error will be thrown?